src/Math-Tests-Core/PMVectorTest.class.st

```diff
@@ -169,6 +169,21 @@ PMVectorTest >> testSubtractScalar [
 	self assert: actual equals: expected.
 ]
 
+{ #category : #test }
+PMVectorTest >> testTake [
```

Comment:
I'd recommend being more specific with the test name here. You could do something like

`testThatTakeYieldsACollection...`

Reply:
I think that the name is good, but this test could be split in two:
```Smalltalk
testTake
	| vector |
	vector := #(1 2 3) asPMVector.
	self assert: (vector take: 2) equals: #(0 1 3) asPMVector.
	self assert: (vector take: 3) equals: #(0 0 1) asPMVector.

testTakeEmpty
	| vector |
	vector := #() asPMVector. 
	self assert: (vector take: 2) equals: #() asPMVector.
```

Reply:
I'd disagree that the name is good because, looking at the code snippet, I can't tell how `vector take: 2` results in `[0, 1, 3]`

Reply:
Other test you can add include:
- when the argument value is more than the dimension of the vector; and
- the argument of take is 0 or negative

What happens in those cases?